File: numberPattern3.ts
/* This code is generating a string of numbers in a grid format with 3 rows and 3 columns. It
initializes variables `raw` and `column` to 3, and `counter` and `string` to 0 and an empty string
respectively. It then uses nested for loops to iterate through each row and column, incrementing the
`counter` variable and appending it to the `string`. After each row is complete, a newline character
is added to the `string`. Finally, the `string` is logged to the console. */
const raw = 3;
const column =3;
let counter = 0
let string = ''
for(let i=1;i<=raw;i++){
   for(let j=column;j>=1;j--){
     counter++;
    string +=counter;
}
string += "\n"
}
console.log(string);

File: numberPattern3.java
/**
 * This Java class prints a number pattern in a 3x3 grid.
 */
class NumberPattern3 {
    public static void main(String[] args) {
        int raw = 3;
        int column =3;
        int count =0;
        String s ="";
        for (int i = 1; i <= raw; i++) {
            for (int j =1; j<= column; j++) {
                count++;
                s +=count;
            }
            s+="\n";
        }
        System.out.println(s);
   }
}